dev/Repeater/ViewManager.cpp

```diff
@@ -592,6 +592,16 @@ winrt::UIElement ViewManager::GetElementFromPinnedElements(int index)
 winrt::UIElement ViewManager::GetElementFromElementFactory(int index)
 {
     // The view generator is the provider of last resort.
+
+    auto itemTemplateFactory = m_owner->ItemTemplateShim();
+    if (!itemTemplateFactory)
+    {
+        // If no ItemTemplate was provided, use a default 
+        auto factory = winrt::XamlReader::Load(L"<DataTemplate xmlns='http://schemas.microsoft.com/winfx/2006/xaml/presentation'><TextBlock Text='{Binding}'/></DataTemplate>").as<winrt::DataTemplate>();
```

Comment:
How about using the new x:Bind instead of the old Binding?

Reply:
Can't use here. There is no code-gen in this case to make x:Bind work.

Reply:
Agree.

Reply:
testing a reply

---
In reply to: [239191559](https://github.com/Microsoft/microsoft-ui-xaml/pull/35#discussion_r239191559) [](ancestors = 239191559)